src/include/ceph_features.h

```diff
@@ -76,6 +76,7 @@
 #define CEPH_FEATURE_OSD_HITSET_GMT (1ULL<<54)
 #define CEPH_FEATURE_HAMMER_0_94_4 (1ULL<<55)
 #define CEPH_FEATURE_NEW_OSDOP_ENCODING   (1ULL<<56) /* New, v7 encoding */
+#define CEPH_FEATURE_BLKIN_TRACING (1ULL<<57) // enabled by ifdef, don't overlap
```

Comment:
@jdurgin okay, thanks!

@vears91 could you please update the value of `CEPH_FEATURE_BLKIN_TRACING` here to the first unused bit you can find?